app/src/main/java/com/github/se/eventradar/model/User.kt

```diff
@@ -8,8 +8,9 @@ data class User(
     val lastName: String,
     val phoneNumber: String,
     val accountStatus: String,
-    val eventsAttendeeList: List<String>,
-    val eventsHostList: List<String>,
+    val eventsAttendeeList: MutableSet<String>,
```

Comment:
To maintain consistency across the application, if you think using MutableSet is the better option here, then we should also consider changing the organiserList and attendeeList properties in the Event class from Set to MutableSet. This adjustment would align these properties with the same mutable approach.